ruby_review_week_four.rb

```diff
@@ -2,36 +2,170 @@
 
 # 1. What is a variable?
 
+# it is a named location in memory to store data via assignment (variable = some_data) that can be later called upon with that name to return what is stored there.
+
 # 2. What Ruby method can you use to make a loop from an array?
 
+# Array.each
+
 # 3. In Ruby, what are booleans?
 
+# a data type that only has two possible values, true and false
+# the result of some condition can also return true or false e.g. "if (10 > 9)"" is the same as writing "if true"
+
 # 4. How do classes relate to objects?
 
+# every variable is an object of a certain class type. in a variable we store an instance of an object. the object is created calling the constructor of a class in some way.
+
 # 5. What is an instance variable?
 
+# in a class you can have a variable that is accessible from anywhere with in an instance of that class, i.e. it's scope is the entire object instance, so any methods within that specific object instance can access it, but not any object of the same class type.
```

Comment:
here you could have mentioned the word `attributes`, in that instance variables are attributes for objects, you are right in what you've said however